src/test/java/org/minions/devfund/richard/moviedatabase/ActorTest.java

```diff
@@ -0,0 +1,51 @@
+package org.minions.devfund.richard.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+
+/**
+ * Test Class.
+ */
+public class ActorTest {
+
+    private Actor actor;
+
+    /**
+     * Before.
+     */
+    @Before
+    public void setUp() {
+        actor = new Actor();
+    }
+
+    /**
+     * Test.
+     */
+    @Test
+    public void testAddMovieAlreadyExist() {
+        final Movie movie1 = new Movie();
+        final Movie movie2 = new Movie();
+        movie1.setName("Movie1");
+        movie2.setName("Movie2");
+        actor.setMovies(movie1);
+        actor.setMovies(movie2);
+        actor.setMovies(movie1);
+        assertEquals(2, actor.getMovies().size());
+        assertEquals("Movie1", actor.getMovies().get(0).getName());
+        assertEquals("Movie2", actor.getMovies().get(1).getName());
+    }
+
+    /**
+     * Test.
+     */
+    @Test
+    public void testEquals() {
+        assertFalse(actor.equals(null));
+        assertFalse(actor.equals(new Movie()));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this call to "equals"; comparisons between unrelated types always return false. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2159)